src/veh_interact.cpp

```diff
@@ -1808,12 +1836,18 @@ void complete_vehicle ()
     case 'o':
         // Only parts that use charges
         if (!is_wrenchable && !is_hand_remove && !is_wheel){
-            if( !crafting_inv.has_items_with_quality( "SAW_M_FINE", 1, 1 ) ) {
+            if( !crafting_inv.has_items_with_quality( "SAW_M_FINE", 1, 1 ) && (is_wood && !crafting_inv.has_items_with_quality("HAMMER", 1, 1))) {
                 tools.push_back(tool_comp("circsaw_off", 20));
                 tools.push_back(tool_comp("oxy_torch", 10));
                 g->u.consume_tools(tools);
             }
         }
+        // Nails survive if pulled out with pliers or a claw hammer. TODO: implement PLY tool quality and random chance based on skill/quality
+        /*if(is_wood && crafting_inv.has_items_with_quality("HAMMER", 1, 1)) {
+            item return_nails("nail");
+            return_nails.charges = 10;
+            g->m.add_item_or_charges( g->u.posx, g->u.posy, return_nails );
+        }*/ //causes runtime errors. not a critical feature; implement with PLY quality.
```

Comment:
PRY, and HAMMER 3 is the claw hammer.  1 is rocks, sledge, war hammer sorts of things.